server/models/handlers/timeslot.js

```diff
@@ -0,0 +1,11 @@
+const timeslotModel = require('../db/Timeslot');
+
+// Fetches timeslot records from the timeslot table 
+async function getPossibleTimeslots(specificStartTime, specificEndTime) {
+
```

Comment:
Hi Peter,
I would suggest removing the blank line on line 5 to reduce unnecessarily long code files.

Reply:
Resolved!